helpers.py

```diff
@@ -1,10 +1,19 @@
 import os
 import logging
+import stdiomask
+import requests
 import subprocess
 from colorlog import ColoredFormatter
 from halo import Halo
 
 
+BUMBLEBEE_ONLINE_API_KEY_FILENAME = 'bumblebee_token'
+BUMBLEBEE_ONLINE_GET_API_KEY_URL = "https://c9o8fm.deta.dev/api-key/new"
+BUMBLEBEE_ONLINE_LOGIN_URL = "https://c9o8fm.deta.dev/auth/jwt/login"
+TEST_URL_BUMBLEBEE_LOGIN = "http://127.0.0.1:8000/api-key/new"
+TEST_URL_GET_BUMBLEBEE_API_KEY = "http://127.0.0.1:8000/api-key/new"
+
```

Comment:
Ahh okay, I see now that by `TEST` you mean `localhost`. Maybe we can call it `LOCAL_...`.
`LOCAL_BUMBLEBEE_ONLINE_LOGIN_URL`, while the prod one is `BUMBLEBEE_ONLINE_LOGIN_URL`.

And then finally, we want the decision as to which of the urls to use to occur dynamically depending on if bee was started in local mode or prod mode.

This can be done in an `env.py` file that checks for OS environment variables, and then we use it as follows:

```python
import env

response = requests.get(env.bumblebee_online_login_url, ...)
```

Reply:
Also, it might be wise to store the bumblebee online base url (https://c9o8fm.deta.dev), in its own variable for when it does change.

We can have one for local and for prod.